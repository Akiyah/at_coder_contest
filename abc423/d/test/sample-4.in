1 10
100 300 5
