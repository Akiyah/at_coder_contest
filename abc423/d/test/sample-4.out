100
200
400
600
650
700
